### ISSUE TITLE
[Bug]:Server not generating nether and infinite-Dark 

### ISSUE THREAD

**Issue by ItsDanyGX** [DarkPacks/SkyFactory-One#128]
### General Info

- [ ] I can reproduce this issue consistently in single-player
- [X] I can reproduce this issue consistently in multi-player
- [X] I have searched for this issue previously and it was either (1) not previously reported, or (2) previously fixed and I am having the same problem.
- [ ] I am crashing and can provide my crash report(s)
- [X] I am using the latest version of the modpack

### Modpack version

1.0.2

### Java version

Java 8 update 273

### Modpack Launcher Used

ATlauncher

### Memory Allocated

6Gb

### Minecraft World Environment

Server

### Misc Client Info

- [ ] Optifine Installed
- [ ] Shaders Installed

### Server Java Version

Java 8 update 311

### Server Operating System

Windows

### Misc Server Info

- [ ] Sponge or Non-Vanilla Forge Server

### Issue Description

The overworld dimension generates perfectly but nether and infinite dark dimensions do not generate, I can enter them but they are plains biome all around and no structures generate besides villages. I've tried to replicate this in singleplayer but it works perfectly, also if I generate the dimension in singleplayer and then put it in the server they work ultil I generate new chunks, wich are generated as plains again.

**Comment by Darkosto** [comment]
I've been trying to replicate this issue and have had no luck. It sounds like server generator settings may not be saving correctly. Do you have a log with some more information to help track this down?

**Comment by ItsDanyGX** [comment]
Sorry for the delay in my response, first of all thank you for your quickness and secondly, I have already fixed it, I don't really know if it was a mistake of mine or from the Launcher itself but I downloaded the server from Curse forge, imported the world (and deleted both nether and infinite dark dimensions) and it generated them perfectly now. From what I can tell the problem is either me, who did something wrong, or from ATLauncher itself (I downloaded the server straight from the launcher all the times I tried to replicate the problem). Anyway thank you and keep it up! it's a very fun pack to play.
